projects/Amlogic/bootloader/mkimage

```diff
@@ -3,28 +3,61 @@
 
 . config/options $1
 
-case "${UBOOT_SYSTEM}" in
-  box|"")
-    DTB="@@DTB_NAME@@"
-    ;;
+DTB="$(${SCRIPTS}/uboot_helper ${PROJECT} ${DEVICE} ${UBOOT_SYSTEM} dtb)"
 
-  *)
-    echo "image: writing u-boot to $(basename $DISK)"
-    dd if="$INSTALL/usr/share/bootloader/u-boot.bin.sd.bin" of="$DISK" conv=fsync,notrunc bs=1 count=112 >"$SAVE_ERROR" 2>&1 || show_error
-    dd if="$INSTALL/usr/share/bootloader/u-boot.bin.sd.bin" of="$DISK" conv=fsync,notrunc bs=512 skip=1 seek=1 >"$SAVE_ERROR" 2>&1 || show_error
+mkimage_uboot(){
+  echo "image: writing u-boot to $(basename $DISK)"
+  dd if="$INSTALL/usr/share/bootloader/u-boot.bin.sd.bin" of="$DISK" conv=fsync,notrunc bs=1 count=112 >"$SAVE_ERROR" 2>&1 || show_error
+  dd if="$INSTALL/usr/share/bootloader/u-boot.bin.sd.bin" of="$DISK" conv=fsync,notrunc bs=512 skip=1 seek=1 >"$SAVE_ERROR" 2>&1 || show_error
+}
 
-    DTB="$(${SCRIPTS}/uboot_helper ${PROJECT} ${DEVICE} ${UBOOT_SYSTEM} dtb)"
-    ;;
-esac
+mkimage_autoscripts(){
+  echo "image: copying autoscript files and device trees"
+  mcopy -s ${RELEASE_DIR}/3rdparty/bootloader/* ::
+  touch "$RELEASE_DIR/3rdparty/bootloader/aml_autoscript.zip"
+}
 
-echo "image: creating uEnv.ini"
+mkimage_uEnv(){
+  echo "image: creating uEnv.ini"
 cat << EOF > "${LE_TMP}/uEnv.ini"
-dtb_name=/dtb/${DTB}
```

Comment:
Is there any reason why the dtb files were moved to the root instead of keeping them in a subfolder? Keeping them in the dtb folder was IMHO a somewhat cleaner approach :thinking: